lighthouse-core/test/audits/byte-efficiency/uses-long-cache-ttl-test.js

```diff
@@ -74,6 +74,22 @@ describe('Cache headers audit', () => {
     });
   });
 
+  it('ignores nonpositive and nonfinite max-age headers', () => {
+    const infinityMaxAgeStringValue = '1'.repeat(400);
+    assert.equal(Number.parseInt(infinityMaxAgeStringValue), Infinity);
+    const networkRecords = [
```

Comment:
so, we wish to ignore all of these records right?

FYI `max-age=blahblah` would not be ignored.

Shouldn't something silly like `max-age=-anything` be surfaced? or is that out of scope for this audit

Reply:
what did the spec say for cacheControl? for `expires` it was anything invalid is treated as no cache lifetime, so if cache-control is the same we should probably surface them as having no caching?